comfy_cli/workflow_print.py

```
@@ -0,0 +1,1126 @@
+        _place_arg(iname, ref if ref is not None else "None", args, extra)
+
+    if extra:
+        inner = ", ".join(f"{json.dumps(k, ensure_ascii=False)}: {v}" for k, v in extra.items())
+        args.append(f"**{{{inner}}}")
+
+    widgets_values = node.get("widgets_values")
+    if isinstance(widgets_values, list) and widgets_values:
+        args.append(f"widgets={py_literal(widgets_values)}")
+
+    warnings.append(f"node {ctx.qualify(nid)}: subgraph definition {type_uuid} missing; printed opaquely")
+    line = f"{binding} = Subgraph[{json.dumps(type_uuid)}]({', '.join(args)})  # {nid} subgraph {type_uuid} definition missing"
```

Pylint (4.0.7)

[convention] Line too long (127/120)

(C0301)